templates/store/_entity-header.html

```diff
@@ -65,10 +65,16 @@ <h1 itemprop="name" class="u-no-margin--bottom entity__title">
         <button class="p-code-snippet__action">Copy to clipboard</button>
       </div>
       <p>
-        <a href="{{ external_urls.gui }}?deploy-target={{ context.entity.id }}"
-          class="p-button--positive row" rel="nofollow">
-          Add to new model
-        </a>
+        {% if "kubernetes" in context.entity.series %}
+          <a class="p-button--neutral row is-disabled" aria-disabled="true">
```

Comment:
according to the vanilla docs this should disable the link, but it doesn't, maybe it will with the new vanilla version in the other PR? @barrymcgee

Reply:
The Vanilla docs reflect latest Vanilla version so I assume this will kick in, yes.